flask/app/decorators.py

```diff
@@ -0,0 +1,21 @@
+from functools import wraps
+from typing import Any
+from logging import error
+from flask import request, abort
+
+
+def validate_json(Validator: Any = None):
+    """ HOF that returns a decorator with optional Validator object (currently not implemented) """
+
+    def wrapper(f):
+        @wraps(f)
+        def validate(*args, **kwargs):
+            """ validate content-type header and run optional Validator on input """
+            if request.headers.get("Content-Type") != "application/json":
+                error("Content-Type is not application/json")
```

Comment:
Does the standard logging library work the same as the logger in the Flask app instance?

Reply:
Evidently not. Seems like we'd need to configure a root logger first (something like [this](https://flask.palletsprojects.com/en/1.1.x/logging/#basic-configuration)). I was concerned about logging having a dependency on the app, since it seemed like overkill to spin up the app context just to unit test a function, but I ended up just removing the logging code from the decorator altogether, since the werkzeug logger will print the `415`.